src/commands/alias/ls.js

```diff
@@ -1,34 +1,42 @@
-//      
 import chalk from 'chalk';
 import ms from 'ms';
 import plural from 'pluralize';
 import table from 'text-table';
-
 import Now from '../../util';
-
 import getAliases from '../../util/alias/get-aliases';
 import getScope from '../../util/get-scope';
 import stamp from '../../util/output/stamp';
 import strlen from '../../util/strlen';
 import wait from '../../util/output/wait';
-                                                                              
 
 export default async function ls(
-  ctx            ,
-  opts                 ,
-  args          ,
-  output        
-)                  {
+  ctx,
+  opts,
+  args,
+  output
+) {
   const { authConfig: { token }, config } = ctx;
   const { currentTeam } = config;
   const { apiUrl } = ctx;
   const { '--debug': debugEnabled } = opts;
-  const { contextName } = await getScope({
-    apiUrl,
-    token,
-    debug: debugEnabled,
-    currentTeam
-  });
+
+  let contextName = null;
+
+  try {
+    ({ contextName } = await getScope({
```

Comment:
This syntax is so messed up 😅, at least I know what it does 😄

Reply:
@timneutkens What's messed up about it? There are definitely languages that implement this better, but this is how to achieve it in JS... 😄

Reply:
Destructuring without keywords is just very weird since all other usages of destructuring have keywords (const/let) lol 😄

Reply:
Nothing wrong in doing it because JS allows it 😋